worldedit-bukkit/adapters/adapter-1_19_4/build.gradle.kts

```diff
@@ -9,6 +9,6 @@ repositories {
 }
 
 dependencies {
-    paperDevBundle("1.19.4-R0.1-20230331.112431-38")
+    paperDevBundle("1.19.4-R0.1-20230412.010331-64")
```

Comment:
Its really important to hard lock the version ?

Reply:
Yes, otherwise we have random compilation issues depending on caching on several levels, which is annoying to debug